main.go

```diff
@@ -103,6 +103,7 @@ func main() {
 		Scheme:             scheme,
 		MetricsBindAddress: metricsAddr,
 		LeaderElection:     enableLeaderElection,
+		LeaderElectionID:   "controller-leader-election-helper-cabpk",
```

Comment:
```suggestion
		LeaderElectionID:   "controller-leader-election-cabpk",
```

I don't believe `helper` adds anything to the name of the configMap so it can likely be omitted, otherwise lgtm